server.js

```diff
@@ -1,20 +1,125 @@
 const express = require("express");
 const cors = require("cors");
+const uuid = require("uuid");
+const moment = require("moment");
+const validator = require("email-validator");
 
 const app = express();
 
 app.use(express.json());
+app.use(express.urlencoded({ extended: true}));
```

Comment:
The `urlencoded` middleware is not needed for this task. You only need it if you post form data, but in this task all data is in JSON format.

Reply:
:+1: